WordPress/Resources/bg.lproj/Localizable.strings

```diff
@@ -33,17 +33,19 @@
 /* Title for button when a site is missing server credentials. %1$@ is a placeholder for the string 'Enter your server credentials'. */
 "%1$@ to fix threats." = "%1$@ to fix threats.";
 
-/* Singular format string for view title displaying the number of post likes. %1$d is the number of likes. */
+/* Singular button title to Like a comment. %1$d is a placeholder for the number of Likes.
+   Singular format string for view title displaying the number of post likes. %1$d is the number of likes. */
 "%1$d Like" = "%1$d Like";
 
-/* Plural format string for view title displaying the number of post likes. %1$d is the number of likes. */
+/* Plural button title to Like a comment. %1$d is a placeholder for the number of Likes.
+   Plural format string for view title displaying the number of post likes. %1$d is the number of likes. */
 "%1$d Likes" = "%1$d Likes";
 
-/* Singular format string for displaying the number of post likes. %1$d is the number of likes. */
-"%1$d blogger likes this." = "%1$d blogger likes this.";
+/* Singular format string for displaying the number of post likes. %1$d is the number of likes. The underscore denotes underline and is not displayed. */
+"%1$d blogger_ likes this." = "%1$d blogger_ likes this.";
 
```

Comment:
Was about to comment on the underscore at the end of `blogger_` feeling odd until I read the comment explaining it. Still seems odd that there's only one `_` (i.e. that there isn't a **pair** of underscores **surrounding** the text to be underlined) so still wanted to highlight this to double-check

Reply:
There is only one because the highlighting starts at the beginning of the string.

Reply:
> There is only one because the highlighting starts at the beginning of the string.  So splitting on `_` in this case provides the correct components. 

@ScoutHarris Well… in English maybe it does. But I don't think it's guaranteed to always be the case in all locales. There might be locales that have this "N bloggers" part of the text in the middle bc grammar and word order in that locale is different than in English. And more commonly, that assumption won't hold for RTL locales, would it?

Reply:
Ah, I see. Coincidentally, more changes are coming to these strings in [#16946](https://github.com/wordpress-mobile/WordPress-iOS/pull/16946). We're still discussing how we want them to look, so I don't want to make too many changes here (conflicts and all that). But I'll take a look and see what I can do for 17.9. Thank you for noticing that!

Reply:
Oh yeah we should definitely not block this PR on this (especially because it needs to land before this weekend to have the release notes translated in time), I was just mentioning this to get addressed later (and 16946 seems a good one for that 👍)

Reply:
Ah, ok, cool. With/after 16946, I'll make sure the `_`s are correct. Thank you sir!